src/init.gradle

```diff
@@ -88,6 +90,44 @@ rootProject {
         }
     }
 
+    def cycloneDxBOMTask = tasks.register("generateCycloneDxBom", org.cyclonedx.gradle.CycloneDxTask) {
+        mustRunAfter(dependencyTree)
+        doFirst {
+            configurations {
```

Comment:
The configuration should probably be created _outside_ of the configuration block, it's unusual (and I'm actually surprised it works) that a configuration is created in a task execution.